docs/en/reference/sql/dql/HAVING_CLAUSE.md

```diff
@@ -16,31 +16,29 @@ SELECT select_expr [,select_expr...] FROM ... GROUP BY ... HAVING having_conditi
 ```
 
 ## Boundary Description
+For the standalone version, `HAVING` is supported in all conditions. For the cluster version, the execution modes which support this clause is shown below.
 
-| SELECT statement elements | state          | directions                                                         |
-| :------------- | ------------- | :----------------------------------------------------------- |
-| HAVING Clause  | Online not supported | Having clause is similar to the Where clause. The Having clause allows you to filter various data after GroupBy, and the Where clause is used to filter records before aggregation. |
+| `SELECT` Statement Elements                                | Offline Mode | Online Preview Mode | Online Request Mode | Note                                                                                                                                                               |
+|:-----------------------------------------------------------|--------------|---------------------|---------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| HAVING Clause                | **``✓``**    |                     |                     | The Having clause is similar to the Where clause. The Having clause filters data after GroupBy, and the Where clause is used to filter records before aggregation. |                                                                                                                                                                                                                                                                                                                                                                              |
 
 ## Example
 
 ### 1. Filter By Aggregation Results After Grouping
```

Comment:
can you help remove those title formatting? just using the bold is enough for example titles

Reply:
> can you help remove those title formatting? just using the bold is enough for example titles

I have improved the format of these example titles both in en and zh versions